clippy_lints/src/declare_clippy_lint.rs

```diff
@@ -0,0 +1,173 @@
+// warn on lints, that are included in `rust-lang/rust`s bootstrap
+#![warn(rust_2018_idioms, unused_lifetimes)]
+
+#[macro_export]
+#[allow(clippy::crate_in_macro_def)]
+macro_rules! declare_clippy_lint {
+    (@
+        // $(#[$($attrss:tt)*])*
+        $(#[doc = $lit:literal])*
+        pub $lint_name:ident,
+        $category:ident,
+        $lintcategory:expr,
+        $desc:literal,
+        $version_expr:expr,
+        $version_lit:literal
+    ) => {
+        rustc_session::declare_tool_lint! {
+        $(#[doc = $lit])*
+        #[clippy::version = $version_lit]
```

Comment:
Indentation is a bit off here

Reply:
Done!

Reply:
The parts in `declare_tool_lint! {` should be indented